dev/release/00-prepare-test.rb

```diff
@@ -269,6 +276,14 @@ def test_update_version_post_tag
                         "+Version: #{@release_version}.9000"],
                      ],
                    },
+                   {
+                     path: "r/NEWS.md",
+                     # Note that these are additions only, no replacement
+                     hunks: [
+                       ["+# arrow 0.14.0.9000",
```

Comment:
Could you use `#{@release_version}.9000` here?

Reply:
Oops, yes, will do.